### ISSUE TITLE
Need prebuilds for Node.js v12

### ISSUE THREAD

**Issue by syuilo** [Automattic/node-canvas#1420]
When `npm i canvas`, got `Cannot open include file: 'cairo.h'` error.

## Your Environment
* Windows 10
* Node.js 12.1.0

**Comment by syuilo** [comment]
Also there is 404 warn:
```
node-pre-gyp WARN Tried to download(404): https://github.com/node-gfx/node-canvas-prebuilt/releases/download/v2.5.0/canvas-v2.5.0-node-v72-win32-unknown-x64.tar.gz
```

**Comment by syuilo** [comment]
Probably related to https://github.com/node-gfx/node-canvas-prebuilt/issues/78

**Comment by LinusU** [comment]
Until precompiled binaries are available, you can use the installation instructions here: https://github.com/Automattic/node-canvas/wiki/Installation%3A-Windows

**Comment by domkh** [comment]
Have you installed this: https://www.cairographics.org/download/ ?

**Comment by syuilo** [comment]
Thank you!
Yeah, I know how to install it manually, and I succeeded in installing it that way.
However, it was inconvenient, so I created this issue.

**Comment by chearon** [comment]
appveyor/ci#2921

I still can't get their `Update-NodeJsInstallation (Get-NodeJsLatestBuild 12)` command working so we just have to wait on that issue for now

**Comment by NightYoshi370** [comment]
> Node.JS 12 is available on AppVeyor environment!

@chearon

**Comment by chearon** [comment]
MSYS issue finally fixed, see node-gfx/node-canvas-prebuilt#78
